Export Analysis Reports/Sdl.Community.ExportAnalysisReports/Helpers/Help.cs

```diff
@@ -188,34 +212,65 @@ public static string GetJsonReportPath(string jsonPath)
 			return string.Empty;
 		}
 
-		public static void CreateDirectory(string path)
+		public void CreateDirectory(string path)
 		{
 			if (!Directory.Exists(path))
 			{
 				Directory.CreateDirectory(path);
 			}
 		}
 
-		private static string GetInstalledStudioShortVersion()
+		public Dictionary<string, bool> AddToDictionary(BindingList<LanguageDetails> languages)
+		{
+			var dictionaryResult = new Dictionary<string, bool>();
+			if (languages != null)
+			{
+				foreach (var item in languages)
+				{
+					dictionaryResult.Add(item.LanguageName, item.IsChecked);
+				}
+			}
+
+			return dictionaryResult;
+		}
+
+		public BindingList<LanguageDetails> AddFromDictionary(BindingList<LanguageDetails> languages, Dictionary<string, bool> languagesDictionary)
+		{
+			if (languagesDictionary != null && !languagesDictionary.Equals(new Dictionary<string, bool>()))
```

Comment:
Were you trying to check languagesDictionary.Count > 0 ?
If not, try to make this more readable.

Reply:
The "!languagesDictionary.Equals(new Dictionary<string, bool>()" validates if the dictionary does not contains null values, it might be the case that elements has null values.